MapApp/Controllers/LocationsController.cs

```diff
@@ -65,6 +68,28 @@ public IActionResult Create()
 			return View();
 		}
 
+        // Posting Content to Page on Facebook
+        private async Task<string> FacebookPublish(Location location)
+        {
+            string uri = "https://graph.facebook.com";
+            string page = "/264600174386513/feed";
+            string message = location.Name + " " + location.Description;
+            string accessToken = "EAADz7RiCz5kBAMv0966BKS7AXtWdMwqEztUjZAWQjwBSDm98p2JCPx9J3Vh8Y5wEUIVySxIevZAJOk39VxqYgVKkWYrQIxjI8uVE9r0WZCfNj5eRzKJD0i8mB44WMBZAJ5nnGMsOz1sHE6htlkutAiT5pKBggZAz9BaTxooeCmQZDZD";
```

Comment:
I think that ``accessToken variable could be placed in different place than inside `FacebookPublish` method..
If other Controllers or Methods would like to use that token?
@ilaypilo There is a place for "consts/defines" in MVC?

Reply:
Also `uri` and `page`

Reply:
done 👍